libsolidity/analysis/ContractLevelChecker.h

```diff
@@ -80,8 +93,24 @@ class ContractLevelChecker
 	void checkLibraryRequirements(ContractDefinition const& _contract);
 	/// Checks base contracts for ABI compatibility
 	void checkBaseABICompatibility(ContractDefinition const& _contract);
+	/// Checks for functions in different base contracts that conflict with each
+	/// other and thus need to be overridden explicitly
+	void checkAmbiguousOverrides(ContractDefinition const& _contract) const;
+	/// Resolves an override list of UserDefinedTypeNames to a list of contracts
+	std::set<ContractDefinition const*> resolveOverrideList(OverrideSpecifier const& _overrides) const;
+
+	void checkModifierOverrides(FunctionSet const& _funcSet, ModifierSet const& _modSet, std::vector<ModifierDefinition const*> _modifiers);
+	void checkOverrideList(FunctionSet const& _funcSet, FunctionDefinition const& _function);
+
+	FunctionSet const& getBaseFunctions(ContractDefinition const* _contract) const;
+	ModifierSet const& getBaseModifiers(ContractDefinition const* _contract) const;
 
 	langutil::ErrorReporter& m_errorReporter;
+
+	/// Map of multisets that contain all overridable functions for the given
```

Comment:
```suggestion
     /// Map of multisets containing all functions which can be overriden for the given contract.
```

Reply:
you really don't like "that" compared to "which" :P